test/unit/matrix/test_distribution.cpp

```diff
@@ -248,392 +393,123 @@ struct ParametersIndices {
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {0, 0}, {2, 2}, {2, 2}, {4, 4}, {3, 4}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {0, 0}, {2, 2}, {0, 0}, {5, 9}, {3, 16}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {0, 0}, {2, 2}, {0, 0}, {6, 10}, {5, 17}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {0, 0}, {2, 2}, {5, 9}, {6, 10}, {2, 1}},
-
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {0, 0}, {7, 6}, {0, 0}, {0, 0}, {0, 0}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {0, 0}, {7, 6}, {0, 0}, {1, 1}, {2, 2}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {0, 0}, {7, 6}, {3, 5}, {3, 5}, {0, 0}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {0, 0}, {7, 6}, {2, 2}, {4, 4}, {3, 4}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {0, 0}, {7, 6}, {0, 0}, {5, 9}, {5, 18}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {0, 0}, {7, 6}, {0, 0}, {6, 10}, {5, 18}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {0, 0}, {7, 6}, {5, 9}, {6, 10}, {0, 0}},
-
-    // additionally non-zero source rank
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {1, 1}, {0, 0}, {0, 0}, {0, 0}, {0, 0}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {1, 1}, {0, 0}, {0, 0}, {1, 1}, {0, 4}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {1, 1}, {0, 0}, {3, 5}, {3, 5}, {0, 0}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {1, 1}, {0, 0}, {2, 2}, {4, 4}, {0, 4}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {1, 1}, {0, 0}, {0, 0}, {5, 9}, {6, 19}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {1, 1}, {0, 0}, {4, 8}, {5, 9}, {3, 3}},
-
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {1, 1}, {2, 2}, {0, 0}, {0, 0}, {0, 0}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {1, 1}, {2, 2}, {0, 0}, {1, 1}, {0, 2}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {1, 1}, {2, 2}, {3, 5}, {3, 5}, {0, 0}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {1, 1}, {2, 2}, {2, 2}, {4, 4}, {0, 4}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {1, 1}, {2, 2}, {0, 0}, {5, 9}, {6, 18}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {1, 1}, {2, 2}, {0, 0}, {6, 10}, {6, 18}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {1, 1}, {2, 2}, {5, 9}, {6, 10}, {0, 0}},
-
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {1, 1}, {7, 6}, {0, 0}, {0, 0}, {0, 0}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {1, 1}, {7, 6}, {0, 0}, {1, 1}, {0, 0}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {1, 1}, {7, 6}, {3, 5}, {3, 5}, {0, 0}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {1, 1}, {7, 6}, {2, 2}, {4, 4}, {3, 4}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {1, 1}, {7, 6}, {0, 0}, {5, 9}, {3, 16}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {1, 1}, {7, 6}, {0, 0}, {6, 10}, {4, 17}},
-    {{15, 35}, {3, 4}, {2, 1}, {3, 2}, {1, 1}, {7, 6}, {5, 9}, {6, 10}, {1, 1}},
-};
-
-TEST(DistributionTest, LocalElementDistanceFromGlobalTile) {
-  for (const auto& test : tests_local_distance_from_global_tile) {
-    Distribution obj(test.size, test.block_size, test.grid_size, test.rank, test.src_rank, test.offset);
-
-    EXPECT_EQ(test.local_element_distance,
-              obj.localElementDistanceFromGlobalTile(test.global_tile_begin, test.global_tile_end));
-  }
-}
-
```

Comment:
Since it's hard to tell from the diff: Did these test cases and tests get merged into other tests, are they missing, or are they no longer needed (covered by other tests)?

Reply:
341-488: Moved to `test_distribution_extensions.cpp`
489-636: The method tested doesn't exists anymore (the 2D method was never used).
         I can refactor it for the 1D method, but I don't want to spend time for
         something that will disappear after the tridiagonal_eigensolver Q4 clean-up.

Reply:
All good, thanks for the explanation! I just wanted to make sure they hadn't been accidentally lost.